### PR TITLE
Merging template updates 2.7.2

### PR DESCRIPTION
**Pull request by d4straub**
Updates the template to 2.7.1
The failing linting is a bug of nf-core tools for single-container-modules.
_edit: update for 2.7.2 see below_

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by d4straub** [comment]
So I attempted to merge also template update 2.7.2 which seems to be a major change as well to this pipeline. Due to the removal of `params.enable_conda` all QIIME2 modules (which have no conda env) need to be handled deferentially. I followed the example of [cellranger](https://github.com/nf-core/modules/blob/f4bbd81636426c4060ce7ff421f4500c5a831e29/modules/nf-core/cellranger/count/main.nf).

**Review by erikrikarddaniel — APPROVED** [review]
One comment, otherwise looks fine.

File: lib/WorkflowAmpliseq.groovy [hunk 1]
```diff
@@ -10,7 +10,7 @@ class WorkflowAmpliseq {
     // Check and validate parameters
     //
     public static void initialise(params, log) {
-        if (params.enable_conda) { log.warn "Conda is enabled (`--enable_conda`), any steps involving QIIME2 are not available. Use a container engine instead of conda to enable all software." }
+        //if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) { log.warn "Conda or mamba is enabled, any steps involving QIIME2 are not available. Use a container engine instead of conda to enable all software." }
```
[the remark this answers is not in the document]
Comment (d4straub) [reply lib/WorkflowAmpliseq.groovy:13]: Thanks for the review. I had trouble with that test and commented it out. I should however activate that again, will attempt this as soon as I can spare some time.

[the remark this answers is not in the document]
Comment (d4straub) [reply lib/WorkflowAmpliseq.groovy]: I actually want able to make the warning work in that file. I had to move it to ampliseq.nf instead. But the importance of that warning warrants this imho.

**Pull request merged**